starport/pkg/nodetime/tsc/tsc.go

```diff
@@ -23,6 +23,9 @@ var (
 				Module:           "es2020",
 				TypeRoots:        []string{filepath.Join(nodeModulesPath, "@types")},
 				SkipLibCheck:     true,
+				// Default to not emitting JS client files.
+				// NB: Will need to be configurable to support generating plain js clients in the future.
+				NoEmit: true,
```

Comment:
Adding this globally also changes the behavior in vuex store generation, I think we should tackle it later and only set it for this typescript client.